.gitmodules

```diff
@@ -46,3 +46,6 @@
 [submodule "Kotlin"]
 	path = Kotlin
 	url = https://github.com/careercup/CtCI-6th-Edition-Kotlin.git
+[submodule "Rust"]
```

Comment:
Not required. Please refer  https://github.com/careercup/CtCI-6th-Edition#adding-a-new-language

Reply:
Please close this PR and instead follow these steps to get your rust submodule added https://github.com/careercup/CtCI-6th-Edition#adding-a-new-language